compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -445,15 +448,43 @@ private void defineFunction(DataInputStream dataInStream) throws IOException {
                     for (int i = 0; i < resourcePathCount; i++) {
                         resourcePath.add(names.fromString(getStringCPEntryValue(dataInStream)));
                     }
+                    
+                    List<Location> resourcePathSegmentPosList = new ArrayList<>(resourcePathCount);
+                    for (int i = 0; i < resourcePathCount; i++) {
+                        resourcePathSegmentPosList.add(readPosition(dataInStream));
+                    }
```

Comment:
Shall we do this in the previous for-loop?

Reply:
Yeah thanx